keyboards/omkbd/ergodash/rev1/keymaps/trainman419/keymap.json

```diff
@@ -0,0 +1,155 @@
+{
+  "version": 1,
+  "notes": "",
+  "documentation": "\"This file is a QMK Configurator export. You can import this at <https://config.qmk.fm>. It can also be used directly with QMK's source code.\n\nTo setup your QMK environment check out the tutorial: <https://docs.qmk.fm/#/newbs>\n\nYou can convert this file to a keymap.c using this command: `qmk json2c {keymap}`\n\nYou can compile this keymap using this command: `qmk compile {keymap}`\"\n",
+  "keyboard": "omkbd/ergodash/rev1",
+  "keymap": "trainman419",
+  "layout": "LAYOUT_4key_2u_inner",
+  "layers": [
+    [
+      "KC_EQL",
+      "KC_1",
+      "KC_2",
+      "KC_3",
+      "KC_4",
+      "KC_5",
+      "KC_MNXT",
+      "LT(1,KC_NO)",
```

Comment:
```suggestion
    MO(1),
```

Reply:
I'm confused about why you're suggesting changes here; this is the keymap file exactly as produced by the online QMK configurator. Can you explain the changes you're suggesting?